search/search_repository.go

```diff
@@ -273,40 +277,60 @@ func parseMap(queryMap map[string]interface{}, q *Query) {
 			q.Name = key
 			q.Value = nil
 		case map[string]interface{}:
-			q.Name = key
-			if v, ok := concreteVal[IN]; ok {
-				q.Name = OR
-				c := &q.Children
-				for _, vl := range v.([]interface{}) {
-					sq := Query{}
-					sq.Name = key
-					t := vl.(string)
-					sq.Value = &t
-					*c = append(*c, sq)
-				}
-			} else if v, ok := concreteVal[EQ]; ok {
-				switch v.(type) {
-				case string:
+			if key != OPTS {
```

Comment:
In order to avoid this deeply nested tree, can we just do the following instead?

```go
if key == OPTS {
  continue
}
```
This would leave the majority of the branches untouched.

Reply:
Done 04aa0e30c312e0f21ce1894185999e69f51beaa7